UnicodeEncodeError with Python 2

```
    def to_str(v):
        if isinstance(v, bool):
            return {True: 'true', False: 'false'}[v]
        elif isinstance(v, list):
            return ','.join(to_str(e) for e in v)
        elif v is None:
            return 'null'
        else:
>           return str(v)
E           UnicodeEncodeError: 'ascii' codec can't encode character u'\u2014' in position 4749: ordinal not in range(128)

.tox/py27/local/lib/python2.7/site-packages/jsone/shared.py:111: UnicodeEncodeError
```

The offending text (with some context) is `u'rm specs \u2014 including'`

--- a/jsone/newsfragments/338.bugfix
+++ b/jsone/newsfragments/338.bugfix
@@ -1,0 +1,1 @@
+Strings containing unicode are now handled correctly by str() on Python 2.

--- a/jsone/shared.py
+++ b/jsone/shared.py
@@ -107,6 +107,8 @@ def to_str(v):
         return ','.join(to_str(e) for e in v)
     elif v is None:
         return 'null'
+    elif isinstance(v, string):
+        return v
     else:
         return str(v)
 

--- a/specification.yml
+++ b/specification.yml
@@ -1773,6 +1773,11 @@ context: {val: 24}
 template: {$eval: 'str(val)'}
 result: '24'
 ---
+title: str (unicode)
+context: {val: "let's make a ☃"}
+template: {$eval: 'str(val)'}
+result: "let's make a ☃"
+---
 title: str({})
 context: {}
 template: {$eval: 'str({})'}

